src/index.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable semi */
+/* eslint-disable no-param-reassign */
+/* eslint-disable arrow-body-style */
+/* eslint-disable no-restricted-globals */
+/* eslint-disable object-curly-newline */
+/* eslint-disable object-curly-spacing */
+/* eslint-disable camelcase */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable no-undef */
+/* eslint-disable no-return-assign */
+/* eslint-disable arrow-parens */
```

Comment:
🤨 You are disabling Eslint rules all over the place 😅... Please don't do that, you should try to stick to the rules provided!